app/Http/Controllers/PokerController.php

```diff
@@ -1,133 +1,133 @@
 
-use Illuminate\Http\Request;
-use Illuminate\Support\Facades\DB;
-use Illuminate\Support\Facades\Broadcast;
-use Illuminate\Support\Facades\Log;
+    use Illuminate\Http\Request;
+    use Illuminate\Support\Facades\DB;
+    use Illuminate\Support\Facades\Broadcast;
+    use Illuminate\Support\Facades\Log;
 
-class PokerController extends Controller
-{
+    class PokerController extends Controller
+    {
 
-    public function createRoom(Request $request){
+        public function createRoom(Request $request){
 
-        $room_id = $this->generateRandomString(); //todo make sure this is unique in db        
+            $room_id = $this->generateRandomString(); //todo make sure this is unique in db        
 
-        $now = date('Y-m-d H:i:s');
-        $id = DB::table('poker_rooms')->insertGetId([
-            'room_id' => $room_id,
-            'password' => $request->password,
-            'is_open' => true,
-            'created_at' => $now,
-            'updated_at' => $now
-        ]);
-
-        $one_week_ago = date("Y-m-d H:i:s", strtotime("-1 week"));
+            $now = date('Y-m-d H:i:s');
+            $id = DB::table('poker_rooms')->insertGetId([
+                'room_id' => $room_id,
+                'password' => $request->password,
+                'is_open' => true,
+                'created_at' => $now,
+                'updated_at' => $now
+            ]);
 
-        DB::table('poker_rooms')->where('created_at', '<=', $one_week_ago)->update([
-            'is_open' => 0
-        ]);
-        return response()->json(['success' => true, 'room_id' => $room_id]);
-    }
+            $one_week_ago = date("Y-m-d H:i:s", strtotime("-1 week"));
 
-    function getData(Request $request){
-        $room = DB::table('poker_rooms')->where('room_id', $request->room_id)->first();
-        $players = DB::table('poker_players')->where('room_id', $room->id)->get();
-        return response()->json(['room' => $room, 'players' => $players]);
-    }
-
-    function getRooms(){
-        $rooms = DB::table('poker_rooms')->get();
+            DB::table('poker_rooms')->where('created_at', '<=', $one_week_ago)->update([
+                'is_open' => 0
+            ]);
+            return response()->json(['success' => true, 'room_id' => $room_id]);
+        }
 
-        foreach($rooms as $room){
-            $room->password = $room->password != null;
+        function getData(Request $request){
+            $room = DB::table('poker_rooms')->where('room_id', $request->room_id)->first();
+            $players = DB::table('poker_players')->where('room_id', $room->id)->get();
+            return response()->json(['room' => $room, 'players' => $players]);
         }
-        return response()->json(['rooms' => $rooms]);
-    }
 
-    function passwordCheck(Request $request){
-        $room = DB::table('poker_rooms')->where('room_id', $request->room_id)->first();
+        function getRooms(){
+            $rooms = DB::table('poker_rooms')->get();
```

Comment:
nit: Keeping scale in mind: there are multiple querries to the DB to get the rooms (ex: lines 35, 55, 63), and more queries = more read capacity consumed = more money spent. 

it would be more efficient to make this call once, store it in a global variable, and then read from that variable (and filter as needed like in line 35).